### ISSUE TITLE
Use postprocessing with cudecamera occured problem

### ISSUE THREAD

**Issue by OliverXH** [pmndrs/postprocessing#365]
here is my code: 
https://codepen.io/OliverXH/pen/jOYRRbX?editors=0011

I'm using the texture rendered by cubecamera as the environment map. When using postprocessing with it, the whole scene gradually becomes brighter and brighter. Is there any solution?

You can see the problem from the online example.

**Comment by vanruesc** [comment]
> When using postprocessing with it, the whole scene gradually becomes brighter and brighter.

I'm unable to reproduce this on my end.

Here's an example that uses a dynamic cube map + postprocessing: https://codesandbox.io/s/stupefied-bush-5kcwvk?file=/src/App.js&runonclick=1

**Comment by OliverXH** [comment]
@vanruesc Thank you for your reply.

And I reproduced my problem in the example you provided

![gif](https://user-images.githubusercontent.com/49978259/164953663-b04ae789-5a54-4d01-a57d-9fda985c5c71.gif)

I made some changes in these places, and the problem appeared

![image](https://user-images.githubusercontent.com/49978259/164953722-9a527159-5909-406f-922b-b32470403761.png)
![image](https://user-images.githubusercontent.com/49978259/164953729-5d1df087-31d6-4837-8a55-237abfe39d63.png)

**Comment by OliverXH** [comment]
I found another problem. I let the cube room rotate, and then the following problems will appear
![bandicam 2022-04-24 15-36-03-215_](https://user-images.githubusercontent.com/49978259/164965825-f17dcc11-b490-4407-ad7f-beca000bb7a5.gif)

**Comment by vanruesc** [comment]
Thanks for following up on this; I can see the issue now. The scene getting brighter at the beginning is a sign of a feedback loop that's formed by rendering to the cube render target while using it as an envmap at the same time. It's actually surprising that there are no feedback loop warnings in the console. Maybe the compiler doesn't catch it with this setup.

The common solution for this is to hide the object(s) that read from the cube map while the cube map is being rendered. But since you're trying to capture the whole scene, this doesn't seem to be possible.

In conclusion, this is a general scene setup error and not a bug in `postprocessing`.

> I found another problem. I let the cube room rotate, and then the following problems will appear

This is due to the fact that the `EffectComposer` disables the renderer's `autoClear` behaviour. You can set that flag to `true` prior to rendering the cube map and then set it back to `false` right after.

**Comment by OliverXH** [comment]
@vanruesc 
Thank you for your reply, which is very helpful to me. 👍👍 I found that it was my mistake to apply the texture rendered by cubecamera globally. At the same time, adjust the autoClear attribute in different positions according to your idea, and the case can work normally. 👍👍👍👍👍👍
